src/dune/coq_rules.ml

```diff
@@ -109,7 +115,12 @@ let setup_rule ~expander ~dir ~cc ~source_rule ~coq_flags ~file_flags
   let dir = Path.build dir in
   let file_flags = file_flags @ [ Command.Args.Dep (Path.build source) ] in
   let cd_arg = Command.Args.As [ "-dyndep"; "opt" ] :: file_flags in
-  let cd_arg = if boot then Command.Args.As ["-boot"] :: cd_arg else cd_arg in
+  let cd_arg =
```

Comment:
I count 78 chars here, is this break intended?

Reply:
It seems that line was not formatted before. Adding spaces inside the `[ .. ]` forces the break.

Reply:
I see, thanks!